src/main/java/de/mediathekview/mlib/filmlisten/FilmToFakeJsonConverter.java

```diff
@@ -331,7 +331,7 @@ private void appendMediaResource(
                 ? durationToString(((Podcast) aMediaResource).getDuration())
                 : "",
             aMediaResource instanceof Podcast
-                ? ((Podcast) aMediaResource).getFileSize(getDefaultResolution(aMediaResource))
+                ? (((Podcast) aMediaResource).getFileSize(getDefaultResolution(aMediaResource))/1024)
```

Comment:
der Code wäre meiner Meinung besser verständlich, wenn die Berechnung in einer Methode getFileSizeInKb gekapselt wäre.

Reply:
Finde ich gut!
Was macht denn mehr Sinn?
Zwei getter mit KB und MB
Oder nur das umbenennen des getters in KB (mit /1024)?

Reply:
Ich denke KB reicht also eher umbenennen